src/main/java/frc/robot/autonomus/TrenchAuto.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.autonomus;
+
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.command_groups.AutoShoot;
+import frc.robot.commands.command_groups.CollectCell;
+import frc.robot.motion_profiling.AutoPath;
+import frc.robot.motion_profiling.FollowPath;
+import frc.robot.subsystems.drivetrain.RotateDrivetrain;
+
+/**
+ * This auto command shoots 3 balls and then goes to collect more cells from the trench run
+ */
+public class TrenchAuto extends SequentialCommandGroup {
+    public TrenchAuto(StartingPose startingPose) {
+        AutoPath autoPath = startingPose == StartingPose.kFacingPowerPort ?
+            AutoPath.FacingPowerPortToTrenchStart : AutoPath.InLineWithTrenchToTrenchStart;
+        addCommands(
+            new AutoShoot(true),
```

Comment:
add wait command... of 0.1 seconds. The shooter detects that the ball has been shot but it takes a little bit of time for the actual ball to fly away from the robot. Use: `new WaitCommand​(double timeout)` 
**OR BETTER:** add to AutoShoot that if it is in auto mode (gets true in the constructor) the AutoShoot need to add `new WaitCommand​(double timeout)` at the end of it...